.github/workflows/inductor.yml

```diff
@@ -18,34 +18,44 @@ concurrency:
 permissions: read-all
 
 jobs:
+  get-label-type:
+    name: get-label-type
+    uses: ./.github/workflows/_runner-determinator.yml
+    with:
+      triggering_actor: ${{ github.triggering_actor }}
+      issue_owner: ${{ github.event.pull_request.user.login || github.event.issue.user.login }}
+      curr_branch: ${{ github.head_ref || github.ref_name }}
```

Comment:
Since these jobs don't trigger on pull request I believe we also need to add:

```
       curr_ref_type: ${{ github.ref_type }}
```

Reply:
Right. Sorry I had a bit of a brain fart and added this bit here instead of leaving it for your PR.  It's okay to leave in though like you said, thank you for having your PR take it from here